src/type-layout.md

```diff
@@ -56,6 +56,17 @@ Most primitives are generally aligned to their size, although this is
 platform-specific behavior. In particular, on x86 u64 and f64 are only
 aligned to 32 bits.
 
+For every primitive numeric type (`u8`, `i8`, `u16`, `i16`, `u32`, `i32`, `u64`,
+`i64`, `u128`, `i128`, `usize`, `isize`, `f32`, and `f64`), `T`, the bit validity
+of `T` is equivalent to the bit validity of `[u8; size_of::<T>()]`. An
+uninitialized byte is not a valid `u8`. A byte at any offset in a reference or
+pointer type may not be a valid `u8` (the semantics of transmuting a reference or
+pointer to a non-pointer type is currently undecided).
```

Comment:
This "may" seems confusing to me. When is it valid or not valid?

Reply:
Those details are undecided as of now.

Reply:
Oh, I see, thanks!